.github/workflows/stage.yml

```diff
@@ -8,9 +8,11 @@ jobs:
     steps:
       - name: git config
         env:
+          GH_APP_ID: ${{ vars.GH_APP_ID }}
           GH_APP_USER: ${{ vars.GH_APP_USER }}
         run: |
           git config --global user.name $GH_APP_USER
+          git config --global user.email $GH_APP_ID+reearth-app[bot]@users.noreply.github.com
```

Comment:
> [!TIP]
> I referred to this for the syntax.


Ref: [How to correctly set a Github App committer to show the app created the commit?](https://stackoverflow.com/questions/66908568/how-to-correctly-set-a-github-app-committer-to-show-the-app-created-the-commit)